src/workerd/api/blob.c++

```diff
@@ -57,12 +57,13 @@ jsg::BufferSource concat(jsg::Lock& js, jsg::Optional<Blob::Bits> maybeBits) {
     size += partSize;
   }
 
-  auto backing = jsg::BackingStore::alloc<v8::ArrayBuffer>(js, size);
+  auto backing = jsg::BackingStore::alloc<v8::ArrayBuffer>(js, size, jsg::Lock::AllocOption::UNINITIALIZED);
   auto result = jsg::BufferSource(js, kj::mv(backing));
 
   if (size == 0) return kj::mv(result);
 
   auto view = result.asArrayPtr();
+  [[maybe_unused]] const size_t originalSize = view.size();
```

Comment:
No need for the `maybe_unused` annotation since the assert immediately below uses it :-)